docs/contributing.rst

```diff
@@ -177,7 +178,8 @@ and re-run the test. It will now pass again:
 .. note::
 
     Please choose a name for the contract file, that is self-explainatory in the sense of what is been tested, e.g. ``double_variable_declaration.sol``.
```

Comment:
Two typos: ``self-explainatory`` => ``self-explanatory`` and ``in the sense of what is *been* tested``.
Either ``of what is being tested`` or ``of what is to be tested``. Or just ``that is self-explanatory.``

Reply:
Or maybe even ``Please choose a name for the contract file, that explains what is being tested.`` or ``that explains what it tests.``.